src/encoder.rs

```diff
@@ -153,6 +153,17 @@ impl<W: Write> Writer<W> {
         }
         self.write_chunk(chunk::IDAT, &zlib.finish()?)
     }
+    
+    pub fn write_to_dpi(&mut self, x: u32, y: u32, unit: bool) -> Result<()>  {
```

Comment:
That design makes it possible to write a `dpi` chunk after an image chunk. However, [the specification](https://www.w3.org/TR/PNG/#5ChunkOrdering) only permits it before the first `IDAT` chunk. This *at least* deserves better documentation. Maybe a better place would be to place it into the `Encoder` and write the `pHYs` information as part of the header in https://github.com/image-rs/image-png/blob/e2c1152f9138d6f441c1356fc7444482bab85a3a/src/encoder.rs#L65 ?

And `unit` should not be a `bool` but a proper enum (with conversion to `u8`). Otherwise, the values leave one guessing which meaning they have.